src/Randomizer.SMZ3.Tracking/AutoTracking/GameService.cs

```diff
@@ -161,12 +176,12 @@ public bool TryHealPlayer()
         /// <returns>False if it is currently unable to give magic to the player</returns>
         public bool TryFillMagic()
         {
-            if (_autoTracker == null || !_autoTracker.IsConnected || _autoTracker.CurrentGame != Game.Zelda || !_autoTracker.IsInSMZ3)
+            if (AutoTracker == null || !AutoTracker.IsConnected || AutoTracker.CurrentGame != Game.Zelda || !AutoTracker.IsInSMZ3)
```

Comment:
See above. Since this is recurring, maybe this is worth putting into a separate property/method?